index.html

```diff
@@ -3,21 +3,240 @@
+                <h3 class="text-center">Featured Speakers</h3>
+                <div class="underline">
+                    <hr>
+                </div>
+            </div>
+            <div class="row">
+                <section class="col-md-6 d-flex f-speaker mb-5">
+                    <div class="img spimage">
+
+                    </div>
+                    <div class="d-flex flex-column ps-3">
+                        <h3></h3>
+                        <p></p>
+                        <p></p>
+                    </div>
+                </section>
+                <section class="col-md-6 d-flex f-speaker mb-5">
+                    <div class="img spimage">
+
+                    </div>
+                    <div class="d-flex flex-column ps-3">
+                        <h3></h3>
+                        <p></p>
+                        <p></p>
+                    </div>
+                </section>
+                <section class="col-md-6 d-flex f-speaker mb-5">
+                    <div class="img spimage">
+
+                    </div>
+                    <div class="d-flex flex-column ps-3">
+                        <h3></h3>
+                        <p></p>
+                        <p></p>
+
+                    </div>
+
+                </section>
+                <section class="col-md-6 d-flex f-speaker mb-5">
+                    <div class="img spimage">
+
+                    </div>
+                    <div class="d-flex flex-column ps-3">
+                        <h3></h3>
+                        <p></p>
+                        <p></p>
+                    </div>
+                </section>
+            </div>
+        </section>
+    </div>
+    <footer>
+        <div class="container-fluid footer-bg partner">
+            <section class="container d-flex flex-column justify-content-center align-items-center">
+                <div class="partner-header d-flex flex-column justify-content-center align-items-center">
+                    <h3 class="partner-title">Partner</h3>
+                    <div class="underline"><hr></div>
+                </div>
+                <ul class="d-flex list-unstyled flex-wrap justify-content-around col-12">
+                    <li><img src="img/JetBlueLogo.png" alt="partner1"></li>
+                    <li><img src="img/logitech_logo.png" alt="partner2"></li>
+                    <li><img src="img/P_G_logo.png" alt="partner3"></li>
+                    <li><img src="img/ypf_logo.png" alt="partner4"></li>
+                    <li><img src="img/P_G_logo.png" alt=""></li>
+                </ul>
+            </section>
+        </div>
+        <section class="footer ">
+            <div class=" footr logitep">
+                <img class="header-img" src="img/work.png" alt="logo">
+                <h5 class="tfooter">Kadoma Community <br> Development</h5>
+                <p class="footerp">2021 creative commons spome rights reserved </p>
+            </div>
+        </section>
```

Comment:
- [ ] On the homepage on mobile screen the footer section should look like the given design with simply the `More` dropdown on the page.

| Original | Yours |
| ------ | ----- |
| <img src="https://user-images.githubusercontent.com/33323077/133800061-4a94b0a3-9978-4b6c-bc5c-c0eba2ef9762.png" height="250px"/> | <img src="https://user-images.githubusercontent.com/33323077/133800154-503a3840-ed9a-462c-8a32-34d3b5ec9378.png" height="250px"/>